controllers/heat_controller.go

```diff
@@ -350,23 +350,23 @@ func (r *HeatReconciler) reconcileNormal(ctx context.Context, instance *heatv1be
 	// Handle service init
 	ctrlResult, err := r.reconcileInit(ctx, instance, helper, serviceLabels)
 	if err != nil {
-		return ctrlResult, err
+		return ctrlResult, fmt.Errorf("Error initializing service: %w", err)
```

Comment:
I think we should use the consistent message in a similar logic.

In HeatAPI/Engine the error message for reconcieInit failure is `fmt.Errorf("Failed to initialize %s: %w", instance.Name, err)` which is different from this one.

Also